bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableIO.java

```diff
@@ -791,17 +754,14 @@ public BufferedMutatorDoFn(CloudBigtableConfiguration config) {
     }
 
     @Setup
-    public synchronized void setup() {
```

Comment:
Please a TODO for re-adding this
Also we should chat about how we can re-add this